python/hail/expr/types.py

```diff
@@ -278,14 +263,9 @@ def _repr(self):
 
 
 class TString(Type):
-    """
-    Hail type corresponding to str.
-
-    .. include:: ../_templates/hailType.rst
-
-    - `expression language documentation <types.html#string>`__
-    - in Python, these are represented natively as Python unicode strings
+    """Hail type for text strings.
 
+    In Python, these are represented as :obj:`unicode` strings.
```

Comment:
In Python 3 (which, AFAIK, is the target python for the release of 0.2), this is ``` :obj:`str` ``` right?